FinniversKit/Sources/Fullscreen/FrontPage/FrontPageView.swift

```diff
@@ -269,6 +293,22 @@ public final class FrontPageView: UIView, BasicFrontPageView {
         
     }
     
+    public func configureFrontpageShelfsWith(_ model: FrontPageShelfViewModel) {
```

Comment:
I think it reads nicer with
`configureFrontPageShelves(with model`
just for consistency